### PR TITLE
Browser: Make middle click and ctrl click tab focus consistent with m…

### PR DESCRIPTION
**Pull request by Coderdreams**
…odern browser behavior

They both create background tabs

### CODE REVIEW

File: Userland/Applications/Browser/Tab.h [hunk 1]
```diff
@@ -8,6 +8,7 @@
 #pragma once
 
 #include "History.h"
+#include "LibWeb/HTML/ActivateTab.h"
```
Comment (linusg) [inline Userland/Applications/Browser/Tab.h:11]: use `<>` includes

File: Userland/Applications/Browser/Tab.cpp [hunk 2]
```diff
@@ -359,8 +362,8 @@ Tab::Tab(BrowserWindow& window)
         m_image_context_menu->popup(screen_position);
     };
 
-    view().on_link_middle_click = [this](auto& href, auto&, auto) {
-        view().on_link_click(href, "_blank", 0);
+    view().on_link_middle_click = [this](auto& href, auto& target, auto) {
+        view().on_link_click(href, target, Mod_Ctrl);
```
Comment (linusg) [inline Userland/Applications/Browser/Tab.cpp:366]: This should probably just invoke `on_tab_open_request` directly.

Comment (Coderdreams) [reply Userland/Applications/Browser/Tab.cpp:366]: I was checking that but when I looked at the: 
"// TODO: maybe activate tabs according to some configuration, this is just normal current browser behavior"
in on_link_click I realized maybe it's better if we keep the link opening routines centralized to configure the behavior in a single place. What do you think?

File: Userland/Applications/Browser/Tab.h (hunk 1 above)
Comment (Coderdreams) [reply Userland/Applications/Browser/Tab.h]: Done

File: Userland/Applications/Browser/Tab.cpp (hunk 2 above)
Comment (ADKaster) [reply Userland/Applications/Browser/Tab.cpp:366]: @Coderdreams hard coding the middle click behavior to be "The same as ctrl+click" is a bit more confusing than saying that it should open the url in the background, with target of `about::blank`.

Comment (Coderdreams) [reply Userland/Applications/Browser/Tab.cpp:366]: @ADKaster but middle-click behavior is the same as ctrl-click in current browsers, it opens tabs in the background, not the same behavior as target "_blank" that activates the tab.

**Comment by Coderdreams** [comment]
Closing since there's no interest in merging these changes,

**Pull request closed without merging**